slingr-stack/gmelius-package
# #1 migration to package gmelius endpoint

Pull-request for issue #14539 created by @agreggio-slingr

## What it does?
Gmelius endpoint migration to packages

## How to test it?

To test this functionality, navigate to 'idea2' -> 'test1' -> 'package' -> 'Marketplace' and install the 'gmelius-test' package. The appropriate credentials have already been configured. You should verify that the token is generated correctly.
Open console and run:
`log(JSON.stringify(pkg.gmelius.functions.me.get()));`


## Implementation notes

N/A

## Deployment notes

There are no deployment notes
